### PR TITLE
checkstyle: update to 10.3.4

### PR DESCRIPTION
**Pull request by muhlba91**

### CODE REVIEW

File: checkstyle.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=8.40,8.41.1,8.45.1,9.0.1,9.1,9.2,9.2.1,10.0,10.1,10.2,10.3.2
-publicVersions=9.3,10.3.3
+archivedVersions=8.40,8.41.1,8.45.1,9.0.1,9.1,9.2,9.2.1,10.0,10.1,10.2,10.3.2,10.3.4
```
Comment (ganncamp) [inline checkstyle.properties:4]: this should be 10.3.3?

**Review by ganncamp — CHANGES_REQUESTED** [review]

**Review by ganncamp — APPROVED** [review]

File: checkstyle.properties [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 
+10.3.4.description=Upgrade to Checkstyle 10.3.4
+10.3.4.sqVersions=[9.0,LATEST]
+10.3.4.date=2022-11-02
+10.3.4.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/10.3.4
+10.3.4.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/10.3.4/checkstyle-sonar-plugin-10.3.4.jar
+
 10.3.3.description=Upgrade to Checkstyle 10.3.3
-10.3.3.sqVersions=[9.0,LATEST]
+10.3.3.sqVersions=[9.0,9,7.*]
```
Comment (ganncamp) [inline checkstyle.properties:17]:
```suggestion
10.3.3.sqVersions=[9.0,9.7.*]
```

**Review by ganncamp — CHANGES_REQUESTED** [review]
One more change. This is what caused the build to fail.

File: checkstyle.properties (hunk 2 above)
Comment (muhlba91) [reply checkstyle.properties]: fixed, thank you! :)

**Review by ganncamp — APPROVED** [review]

**Pull request merged**